webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5266,6 +5266,35 @@ var userOverride = {
                         $mck_box_form.addClass('mck-text-req');
                         return false;
                     }
+                    if (
+                        $mck_text_box.data('validation') &&
+                        $mck_text_box.data('validation') != ''
+                    ) {
+                        var regexForm = $mck_text_box.data(
+                            'validation'
+                        );
+                        regexForm = new RegExp(regexForm);
+                        if (!regexForm.test(message)) {
+                            document.getElementById(
+                                'mck-form-field-error-alert'
+                            ).innerHTML = $mck_text_box.data(
+                                'ErrorMessage'
+                            );
+                            $applozic('#mck-form-field-error-alert-box')
+                                .removeClass('n-vis')
+                                .addClass('vis');
+                            setTimeout(function () {
+                                $applozic('#mck-form-field-error-alert-box')
+                                    .removeClass('vis')
+                                    .addClass('n-vis');
+                            }, 2000);
+                            return false;
+                        }
+                        $mck_text_box.attr('data-text', 'Type your message...');
```

Comment:
we should be picking it up from the labels

Reply:
@aryankirsali by labels, means MCK_LABELS https://docs.kommunicate.io/docs/web-localization#chat-localization-setup